### ISSUE TITLE
FireBase sendEmailVerification in flutter plugin

### ISSUE THREAD

**Issue by LiveRock** [flutter/flutter#17767]
Any news or update that this will be implemented?

I can't seem to get the method described in 
https://github.com/flutter/plugins/pull/529

In my pubspec.yaml:
dependencies:
  flutter:
    sdk: flutter
  firebase_auth: "^0.5.10"

**Comment by aqwert** [comment]
The `sendEmailVerification` method is on the FirebaseUser object. 

```
    var user = await authService.currentUser();
    await user. sendEmailVerification();
```

If that is what you are referring to

**Comment by LiveRock** [comment]
Thank you. This is exactly what I needed. Where can I find the docs for the firebase auth plugin?

**Comment by aqwert** [comment]
Well, I have not seen any official documentation. I tend to look at the source code and commit history to see what is added

**Comment by zoechi** [comment]
That should be implemented flutter/plugins#529
Dup of #16776, #12459
